src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateMethod/GenerateMethodTests.vb

```diff
@@ -3902,7 +3902,7 @@ Module M
         Dim x As Boolean = Await [|F|]().ConfigureAwait(False)
     End Sub 
 End Module",
-"Imports System
+"Imports System 
```

Comment:
why did this change?

Reply:
I'm not sure. Will investigate tomorrow.